src/Components/3DV/SceneView.tsx

```diff
@@ -543,7 +547,7 @@ function SceneView(props: ISceneViewProps, ref) {
         resetCamera: (meshIds: string[]) => {
             if (meshIds?.length) {
                 createOrZoomCamera(meshIds);
-            } else {
+            } else if (sceneRef.current?.meshes) {
```

Comment:
whats the story with this change?

Reply:
if we don't have any 3d model loaded, the reset button in the camera control was throwing error as we click on it since the 3d scene is not loaded:
![image](https://user-images.githubusercontent.com/45217314/171919461-184fd2b6-832a-4950-86a8-33f572f43260.png)